tests/main.test.ts

```diff
@@ -94,7 +95,7 @@ for (const [name, version] of testedPackageManagers) {
       await expect(runCli(cwd, [name, `--version`])).resolves.toMatchObject({
         exitCode: 0,
         stderr: ``,
-        stdout: `${version.split(`+`, 1)[0]}\n`,
+        stdout: expect.stringMatching(new RegExp(version.split(`+`, 1)[0])),
```

Comment:
You cannot call `RegExp` constructor on such string that contains char with special meaning when used in regex context. Why doesn't the current test suffice?

Reply:
![image](https://github.com/nodejs/corepack/assets/5574625/baf3b6c5-3bd0-4946-918a-ccc4ca3e3a70)

We will additionally print some current environment and context information in cnpm -v. It is not possible to make accurate assertions through stdout.

Reply:
You can use `expect.stringContaining` which doesn't require the use of `RegExp`.